compiler/optimizer/VPHandlers.cpp

```diff
@@ -4115,72 +4089,6 @@ TR::Node *constrainCheckcast(OMR::ValuePropagation *vp, TR::Node *node)
       vp->addBlockConstraint(child, TR::VPNonNullObject::create(vp));
       }
 
-   // Mark objects for tenured alignment
```

Comment:
This seems different - can you justify the removal please?

Reply:
For tenured alignment, this code block will eventually set a bit in the instance hot field descriptor that will have the similar effect to marking a hot field. This bit marking is handled by Scavenger (gencon) or CopyForwardScheme (balanced) the same as when a hot field is marked and performs the object alignment and padding operations. This code will have no effect if scavengerAlignHotFields option is removed which is a primary purpose for this PR. 